### PR TITLE
ipatests: Test to check IPACAChainExpirationCheck when IPA cacrt is renamed

### PR DESCRIPTION
**Pull request by menonsudhir**
This testcase checks that ERROR message is displayed by IPACAChainExpirationCheck when ipa ca crt file is renamed.

### CODE REVIEW

**Comment by rcritten** [comment]
./ipatests/test_integration/test_ipahealthcheck.py:741:56: W291 trailing whitespace
./ipatests/test_integration/test_ipahealthcheck.py:744:81: E501 line too long (81 > 80 characters)
./ipatests/test_integration/test_ipahealthcheck.py:745:14: E111 indentation is not a multiple of four
./ipatests/test_integration/test_ipahealthcheck.py:745:14: E113 unexpected indentation
./ipatests/test_integration/test_ipahealthcheck.py:745:81: E501 line too long (81 > 80 characters)
./ipatests/test_integration/test_ipahealthcheck.py:753:17: E117 over-indented

File: ipatests/test_integration/test_ipahealthcheck.py [hunk 1]
```diff
@@ -723,6 +723,41 @@ def test_run_with_stopped_master(self, ipactl):
...
+    def move_ipa_ca_crt(self):
+        """
+        Fixture to move ipa_ca_crt and revert
+        """
+        self.master.run_command(
+            ["mv", paths.IPA_CA_CRT, paths.IPA_CA_CRT + ".old"]
+        )
+        yield
+        self.master.run_command(
+            ["mv", paths.IPA_CA_CRT + ".old", paths.IPA_CA_CRT]
```
Comment (mrizwan93) [inline ipatests/test_integration/test_ipahealthcheck.py:736]: Can we remove concatenation?

Comment (menonsudhir) [reply ipatests/test_integration/test_ipahealthcheck.py:736]: @mrizwan93 removed the concatenation.

**Comment by rcritten** [comment]
master:

* fcc99813f58825efb093f477544076e272c62ea9 ipatests: Test to check IPACAChainExpirationCheck when IPA cacrt is renamed

**Pull request closed without merging**